### ISSUE TITLE
Safari and negative look-behinds

### ISSUE THREAD

**Issue by hildjj** [hildjj/node-inspect-extracted#18]
Safari doesn't support negative look-behind assertions of the form `(?<!y)x`.  That's pretty awful.

It makes [this line](https://github.com/hildjj/node-inspect-extracted/blob/6cf5c1fff1ca3ca44a6be93834b71630f2c3cbc2/src/inspect.js#L205) fail to parse:

```js
const strEscapeSequencesRegExp = /[\x00-\x1f\x27\x5c\x7f-\x9f]|[\ud800-\udbff](?![\udc00-\udfff])|(?<![\ud800-\udbff])[\udc00-\udfff]/;
```

which makes the global variable `util` not available. 

See https://github.com/peggyjs/peggy/issues/371

**Comment by hildjj** [comment]
This will be fixed in Safari 16.4.  I wonder if Peggy can work around it until then.

**Comment by hildjj** [comment]
I decided to just go ahead and fix this, even though the fix was u-g-l-y.
